src/api/roles/client/client.py

```
@@ -190,6 +205,27 @@ def me(db = Depends(get_session), acc: Account = Depends(get_client_account)):
         last_recorded_height=height,
     )
 
+
+@router.get("/coach_availability/{coach_id}", response_model=CoachAvailabilityResponse)
+def get_coach_availability_for_client(coach_id: int, db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)